### ISSUE TITLE
jshomes.org "Find out more" 404

### ISSUE THREAD

**Issue by davidhund** [ericelliott/jshomes.org#3]
FYI: I'm not sure the jshomes.org site is 'live' yet, but I noticed the "Find out more" button results in a 404.

The `href` is set to `clkn/http://learn-javascript.ericelliott.me/prelaunch/` and results in `https://echoentertainment.club/prelaunch/`, which returns a 404...

**Comment by ericelliott** [comment]
Yeah, jshomes.org is not "live". That's just a placeholder. Thanks for bringing the broken link to my attention, though. =)
